installer/Migration/V5_5_0/lang-string/claim.yaml

```diff
@@ -56,3 +56,15 @@ langStrings:
   -
     value: "Pay"
     unitId: pay
+  -
+    value: "Add Attachment"
+    unitId: add_attachment
```

Comment:
"add_attachment" already exists under general. please don't duplicate the langstrings as it will cause build failures.

Reply:
i checked. but not available in general

Reply:
https://github.com/orangehrm/orangehrm/blob/main/installer/Migration/V5_0_0/lang-string/general.yaml#L385